From 22e9162037637bba1b1f60aac7c707dd1619c063 Mon Sep 17 00:00:00 2001
From: ubuntu <ubuntu@wmdepos.aus.stglabs.ibm.com>
Date: Mon, 20 Feb 2017 15:37:59 -0600
Subject: [PATCH] genesis: keep local repo after genesis installation

Genesis will pxe boot and install ubuntu on target clusters.  At the
end of the install, cobbler will replace the local repo and point
the machines to a global target.  However, if the cluster is not on
an external network, we can't install any additional packages.

The patch will prevent genesis from pointing clusters to a new repo
---
 os_images/config/ubuntu-14.04.4-server-amd64.seed         | 2 +-
 os_images/config/ubuntu-14.04.4-server-amd64.sm.seed      | 2 +-
 os_images/config/ubuntu-14.04.4-server-ppc64el.seed       | 2 +-
 os_images/config/ubuntu-14.04.5-server-amd64.seed         | 2 +-
 os_images/config/ubuntu-14.04.5-server-amd64.sm.seed      | 2 +-
 os_images/config/ubuntu-14.04.5-server-ppc64el.seed       | 2 +-
 os_images/config/ubuntu-16.04.1-server-amd64.seed         | 2 +-
 os_images/config/ubuntu-16.04.1-server-amd64.sm.seed      | 2 +-
 os_images/config/ubuntu-16.04.1-server-ppc64el.raid1.seed | 2 +-
 os_images/config/ubuntu-16.04.1-server-ppc64el.seed       | 2 +-
 10 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/os_images/config/ubuntu-14.04.4-server-amd64.seed b/os_images/config/ubuntu-14.04.4-server-amd64.seed
index 07fa84a..0dd3975 100644
--- a/os_images/config/ubuntu-14.04.4-server-amd64.seed
+++ b/os_images/config/ubuntu-14.04.4-server-amd64.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-14.04.4-server-amd64.sm.seed b/os_images/config/ubuntu-14.04.4-server-amd64.sm.seed
index 07fa84a..0dd3975 100644
--- a/os_images/config/ubuntu-14.04.4-server-amd64.sm.seed
+++ b/os_images/config/ubuntu-14.04.4-server-amd64.sm.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-14.04.4-server-ppc64el.seed b/os_images/config/ubuntu-14.04.4-server-ppc64el.seed
index bac7393..982c56b 100644
--- a/os_images/config/ubuntu-14.04.4-server-ppc64el.seed
+++ b/os_images/config/ubuntu-14.04.4-server-ppc64el.seed
@@ -61,7 +61,7 @@ d-i pkgsel/update-policy select none
 d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-ppc64el/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.4-server-ppc64el/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-14.04.5-server-amd64.seed b/os_images/config/ubuntu-14.04.5-server-amd64.seed
index f715050..7bfd389 100644
--- a/os_images/config/ubuntu-14.04.5-server-amd64.seed
+++ b/os_images/config/ubuntu-14.04.5-server-amd64.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-14.04.5-server-amd64.sm.seed b/os_images/config/ubuntu-14.04.5-server-amd64.sm.seed
index f715050..7bfd389 100644
--- a/os_images/config/ubuntu-14.04.5-server-amd64.sm.seed
+++ b/os_images/config/ubuntu-14.04.5-server-amd64.sm.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-14.04.5-server-ppc64el.seed b/os_images/config/ubuntu-14.04.5-server-ppc64el.seed
index e4daf12..0fbbeb4 100644
--- a/os_images/config/ubuntu-14.04.5-server-ppc64el.seed
+++ b/os_images/config/ubuntu-14.04.5-server-ppc64el.seed
@@ -61,7 +61,7 @@ d-i pkgsel/update-policy select none
 d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-ppc64el/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-14.04.5-server-ppc64el/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-16.04.1-server-amd64.seed b/os_images/config/ubuntu-16.04.1-server-amd64.seed
index 3657151..0fa5ab5 100644
--- a/os_images/config/ubuntu-16.04.1-server-amd64.seed
+++ b/os_images/config/ubuntu-16.04.1-server-amd64.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-16.04.1-server-amd64.sm.seed b/os_images/config/ubuntu-16.04.1-server-amd64.sm.seed
index 3657151..0fa5ab5 100644
--- a/os_images/config/ubuntu-16.04.1-server-amd64.sm.seed
+++ b/os_images/config/ubuntu-16.04.1-server-amd64.sm.seed
@@ -62,7 +62,7 @@ d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i grub-installer/bootdev string $getVar('install_disk', '/dev/sda')
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-amd64/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-amd64/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-16.04.1-server-ppc64el.raid1.seed b/os_images/config/ubuntu-16.04.1-server-ppc64el.raid1.seed
index 035464d..84b3e92 100644
--- a/os_images/config/ubuntu-16.04.1-server-ppc64el.raid1.seed
+++ b/os_images/config/ubuntu-16.04.1-server-ppc64el.raid1.seed
@@ -98,7 +98,7 @@ d-i pkgsel/update-policy select none
 d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-ppc64el/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-ppc64el/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
diff --git a/os_images/config/ubuntu-16.04.1-server-ppc64el.seed b/os_images/config/ubuntu-16.04.1-server-ppc64el.seed
index f6c1ae1..79bb598 100644
--- a/os_images/config/ubuntu-16.04.1-server-ppc64el.seed
+++ b/os_images/config/ubuntu-16.04.1-server-ppc64el.seed
@@ -61,7 +61,7 @@ d-i pkgsel/update-policy select none
 d-i grub-installer/only_debian boolean false
 d-i grub-installer/with_other_os boolean false
 d-i preseed/late_command string \
-in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-ppc64el/sources.list -O /etc/apt/sources.list; \
+##in-target /usr/bin/wget http://$http_server/ubuntu-16.04.1-server-ppc64el/sources.list -O /etc/apt/sources.list; \
 in-target mkdir /root/.ssh; \
 in-target /bin/chmod 700 /root/.ssh; \
 in-target /usr/bin/wget http://$http_server/authorized_keys -O /root/.ssh/authorized_keys; \
-- 
2.7.4

